template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/oozie/OozieRoleConfigProvider.java

```diff
@@ -25,18 +27,28 @@
 
     private static final String OOZIE_DATABASE_PASSWORD = "oozie_database_password";
 
+    private static final String OOZIE_PLUGINS_LIST = "oozie_plugins_list";
+
+    private static final String OOZIE_HA_PLUGINS = "org.apache.oozie.service.ZKUUIDService,"
+        + "org.apache.oozie.service.ZKLocksService,"
+        + "org.apache.oozie.service.ZKXLogStreamingService,"
+        + "org.apache.oozie.service.ZKJobsConcurrencyService";
+
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
         switch (roleType) {
             case OozieRoles.OOZIE_SERVER:
                 RdsView oozieRdsView = getRdsView(source);
-                return List.of(
-                        config(OOZIE_DATABASE_HOST, oozieRdsView.getHost()),
-                        config(OOZIE_DATABASE_NAME, oozieRdsView.getDatabaseName()),
-                        config(OOZIE_DATABASE_TYPE, oozieRdsView.getSubprotocol()),
-                        config(OOZIE_DATABASE_USER, oozieRdsView.getConnectionUserName()),
-                        config(OOZIE_DATABASE_PASSWORD, oozieRdsView.getConnectionPassword())
-                );
+                List<ApiClusterTemplateConfig> config = new ArrayList<>(List.of(
+                    config(OOZIE_DATABASE_HOST, oozieRdsView.getHost()),
+                    config(OOZIE_DATABASE_NAME, oozieRdsView.getDatabaseName()),
+                    config(OOZIE_DATABASE_TYPE, oozieRdsView.getSubprotocol()),
+                    config(OOZIE_DATABASE_USER, oozieRdsView.getConnectionUserName()),
+                    config(OOZIE_DATABASE_PASSWORD, oozieRdsView.getConnectionPassword())));
+                if (isOozieHA(source)) {
+                    config.add(config(OOZIE_PLUGINS_LIST, OOZIE_HA_PLUGINS));
```

Comment:
Have no idea what this is. Assuming it is needed :)

Reply:
Looks like Oozie uses ZK for coordination in HA mode. This plugin list enables that coordination.